packages/common/src/interfaces/excelCopyBufferOption.interface.ts

```diff
@@ -16,6 +16,13 @@ export interface ExcelCopyBufferOption<T = any> {
   /** defaults to "copy-manager", sets the layer key for setting css values of copied cells. */
   copiedCellStyleLayerKey?: string;
 
+  /**
+    * should copy the cells value on copy shortcut even when the editor is opened
+    *
+    * **NOTE**: affects only the default {@link ExcelCopyBufferOption#dataItemColumnValueExtractor}
+    * */
+  copyActiveEditorCell?: boolean;
```

Comment:
would there be a reason to NOT do that though? Do we really need to add this if it's expected to always do that? I remember that SlickGrid will first try to commit if an editor is opened before continuing, so I wonder if we need this extra arg?

Reply:
its a breaking change and we just had one recently ;)

besides, the reason for this could also be that it interfers with the original browser ctrl+c within the editor itself. e.g if it were a text editor and you merely select a part of it would you expect the part or the whole cell being copied?

Reply:
I still don't understand why that is a breaking change? As for the browser ctrl+c, I assume that we have a preventDefault on the even that will not bubble up to the browser (if we don't, then we really should) and so it shouldn't interfere with the browser one

Reply:
not sure what you mean but the default ctrl+c can never happen if there is that `slickCellExternalCopyManager.handleKeyDown` in place that circumvents and overrides the clipboard. And yep it does behave this way with my mentioned feature. So no matter whether you're in the cell or the editor it will always copy the cells content via `dataItemColumnValueExtractor`. As such it definitely feels like a breaking change for me if introduced as default.

Reply:
well ok, it doesn't matter much to me since I'm barely using that feature, so go ahead if you want a flag.

Reply:
the amount of feature toggles is increasing indeed. 😅